backend/src/User/user.service.ts

```diff
@@ -37,6 +39,59 @@ class Service {
         };
     }
 
+    public async sendForgotPassword(email: string) {
+        const user = await UserModel.findByEmailAuth(email);
+        if (!user) {
+            return { status: false, error: 'Email does not match any account.' };
+        }
+
+        // Create token that expires in 1 hour
+        const token = crypto.randomBytes(20).toString('hex');
```

Comment:
What is the difference between   crypto and jwt? If crypto is better should we use it for all tokens?

Reply:
Here, I only create a random string of 20 character, it is not a jwt token, it does not have information or expiry date

Reply:
Should we use jwt instead since we need an expiry date?

Reply:
No, that's the use of a jwt

Reply:
It's less computation since you will only need to use a special token to access the reset password route. That way we don't need to store anything in the database